### PR TITLE
Add the HuggingFace Text Classification Notebook

### PR DESCRIPTION
**Pull request by singh-git10**
*Issue #, if available:*

*Description of changes:*
The notebook for the HuggingFace Text Classification 

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [] I have tested my notebook(s) and ensured it runs end-to-end
- [] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by aaronmarkham** [comment]
Please add this to one of the .rst files so it shows up on the [website](https://sagemaker-examples.readthedocs.io/en/latest/).

**Review by aaronmarkham — CHANGES_REQUESTED** [review]
Please add this to an .rst file.

**Review by atqy — APPROVED** [review]

File: introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1189 @@
+{
```
Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>SageMaker fine-tuned models -&gt; JumpStart fine-tuned models</p><p>Please remove "The inference on the dropdown models can be run in the network isolation as well. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <h3>and available -&gt; and is available</h3><h3>please remove "for inference".</h3><h3>Let's add a title to the section, even without numbering, and then put the current text without bold.</h3><p>Please add: Inference on the models listed in the dropdown menu can be run in network isolation [https://docs.aws.amazon.com/sagemaker/latest/dg/mkt-algo-model-internet-free.html] under VPC settings. However, when running inference on a model specified through HF_MODEL_ID, VPC settings with network isolation will not work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>No need to put it in bold.</p><h3>Please add: , and specify id of any model available in HugginFace <a href="https://huggingface.co/models?pipeline_tag=fill-mask&amp;sort=downloads" target="_blank">Fill-Mask</a> and <a href="https://huggingface.co/models?pipeline_tag=text-classification&amp;sort=downloads" target="_blank">Text-Classification</a>, in the HF_MODEL_ID variable below.</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.    # Retrieve the pre-trained model tarball to further fine-tune
</pre><p>Add, if model_id != "huggingface-tc-models":</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.        model_uri=train_model_uri,
</pre><p>if model_id != "huggingface-tc-models" else None</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Run Batch Transform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Prepare data for Batch Transform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <h3>Deploy model for Batch Transform job</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (khetan2) [inline introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <h3>Compare predictions with the ground truth</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: Done

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p><p> </p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

Comment (singh-git10) [reply introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb:1]: <p>Done</p>

**Comment by singh-git10** [comment]
Addressed all the comments
